packages/Search/examples/point_clouds/point_clouds.hpp

```diff
@@ -0,0 +1,209 @@
+/****************************************************************************
+ * Copyright (c) 2012-2018 by the DataTransferKit authors                   *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the DataTransferKit library. DataTransferKit is     *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <Kokkos_DefaultNode.hpp>
```

Comment:
Why did you include this header from Kokkos rather than `Kokkos_View.hpp`

Reply:
It's from a previous version. I forgot to update it. Will change.